build_tools/scripts/git/update_runtime_submodules.sh

```diff
@@ -10,20 +10,7 @@
 
 set -xeuo pipefail
 
-RUNTIME_SUBMODULES=(
-    "third_party/benchmark"
-    "third_party/cpuinfo"
-    "third_party/flatcc"
-    "third_party/googletest"
-    "third_party/liburing"
-    "third_party/libyaml"
-    "third_party/musl"
-    "third_party/spirv_cross"
-    "third_party/spirv_headers"
-    "third_party/tracy"
-    "third_party/vulkan_headers"
-    "third_party/vulkan_memory_allocator"
-    "third_party/webgpu-headers"
-)
+readarray -t RUNTIME_SUBMODULES < \
+  "$(dirname $(realpath $0))/runtime_submodules.txt"
```

Comment:
Please use the same SCRIPT_DIR incantation we use elsewhere

```suggestion
SCRIPT_DIR="$(dirname -- "$( readlink -f -- "$0"; )")";

readarray -t RUNTIME_SUBMODULES < "${SCRIPT_DIR}/runtime_submodules.txt"
```

Reply:
Done.